source/clear-linux/guides/maintenance/kernel-modules.rst

```diff
@@ -0,0 +1,175 @@
+-------------------
+
+#. From a |CL| system, ensure you are running the *native* kernel. 
+   Currently only the native kernel is enabled to build and load
+   out-of-tree modules.
+
+   .. code-block:: bash
+
+      $ uname -r
+      4.XX.YY-ZZZZ.native
+      # Ensure *.native* is in the kernel name
+
+#. Install the `linux-dev` bundle to obtain the kernel headers, which are
+   required for compiling kernel modules
+
+   .. code-block:: bash
+
+      sudo swupd bundle-add linux-dev
+
+#. Follow instructions from the kernel module source code to compile the 
+   kernel module
+
+
+Load kernel module
+------------------
+
+#. Disable Secure Boot in your system's UEFI settings, if you have enabled
+   it. The loading of new out-of-tree modules modifies the signatures Secure
+   Boot relies on for trust. 
+
+
+#. Disable signature checking for the kernel by modifying the kernel boot 
+   parameters and reboot the system. 
+
+   All kernel modules from |CL| have been signed to enforce kernel security. 
+   However, out-of-tree modules break this chain of trust so this mechanism 
+   needs to be disabled.
+  
+   .. code-block:: bash
+
+      sudo mkdir -p /etc/kernel/cmdline.d
+      echo "module.sig_unenforce" | sudo tee /etc/kernel/cmdline.d/allow-unsigned-modules.conf
+
+#. Update the boot manager and reboot the system to implement the changed 
+   kernel parameters.
+
+   .. code-block:: bash
+
+        sudo clr-boot-manager update
+        sudo reboot
+
+   .. note::
+
+      :command:`clr-boot-manager update` does not return any
+      console output if successful.
+
+   
+#. After rebooting, out-of-tree modules can be manually loaded with 
+   :command:`insmod`. 
+
+   .. code-block:: bash
+
+      sudo insmod ${path_to_module}
+
+
+Optional: Use `modprobe` to specify module options and aliases
+--------------------------------------------------------------
+
+Use :command:`modprobe` to load a module and set options.  
+
+Because :command:`modprobe` can add or remove more than one module, due to 
+modules having dependencies, a method of specifying what options are 
+to be used with individual modules is useful. This can be done with 
+configuration files under the :file:`/etc/modprobe.d` directory. 
+
+.. code-block:: bash
+
+   sudo mkdir /etc/modprobe.d
+
+All files underneath the :file:`/etc/modprobe.d` directory 
+which end with the :file:`.conf` extension specify module options to use when
+loading. This can also be used to create convenient aliases for modules or 
+they can override the normal loading behavior altogether for those with 
+special requirements. 
+
+You can find more info on module loading in the modprobe.d manual page:
+
+.. code-block:: bash
+
+   man modprobe.d
+
+Optional: Configure kernel modules to load at boot
+--------------------------------------------------
+
+The :file:`/etc/modules-load.d` configuration directory can be used to 
+specify kernel modules that should be automatically loaded at boot.
```

Comment:
Suggest rewriting in active voice like the first Optional item. Something like: 
Create a :file:`/etc/modules-load.d` configuration directory to specify kernel modules to automatically load at boot.